### DISCUSSION TITLE
Damian is not redirect

### DISCUSSION THREAD

**Discussion by aryasoni98** [vercel/community#3044] [Help] [answered]
### Problem Description

Custom domain is verified but not redirect .

<img width="880" alt="Screenshot 2023-06-25 at 12 53 03 PM" src="https://github.com/vercel/vercel/assets/18515597/3cc24f61-0f94-408f-995d-3ac031de2619">

**Comment by amyegan** [comment, marked as answer]
Hi @aryasoni98. From the headers on the redirect, it looks like the domain is using Cloudflare. You may need to modify your configuration to stop the redirect loop.

We have guides about [using a Cloudflare domain with Vercel](https://vercel.com/guides/using-cloudflare-with-vercel) and [using a proxy on top of a Vercel Deployment](https://vercel.com/guides/can-i-use-a-proxy-on-top-of-my-vercel-deployment) which usually help. You can also get Cloudflare-specific help by reaching out to [Cloudflare support](https://support.cloudflare.com/hc/en-us).
